README.md

```diff
@@ -14,48 +14,48 @@ In using the Visual Studio Code extension, you have the advantage of relying on
+- For Snyk Open Source, the VS Code extension supports all the languages and package managers supported by Snyk Open Source and the CLI. See the full [list in the docs](https://docs.snyk.io/products/snyk-open-source/language-and-package-manager-support).
+- For Snyk Code, the VS Code extension supports all the [languages and frameworks supported by Snyk Code](https://docs.snyk.io/products/snyk-code/snyk-code-language-and-framework-support#language-support-with-snyk-code-ai-engine).
 
 ## Supported operating systems and architecture
 
 You can use the Snyk Visual Studio Code extension in the following environments:
 
-* Linux: AMD64 and ARM64
-* Windows: 386 and AMD64
-* MacOS: AMD64 and ARM64
+- Linux: AMD64 and ARM64
+- Windows: 386 and AMD64
+- MacOS: AMD64 and ARM64
 
 ## Install the extension
 
 The Snyk Visual Studio Code extension is available for installation on the [Visual Studio code marketplace](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner).
 
 Follow these steps to install:
 
-* Open the settings or preferences in your IDE.
-* Navigate to the [Snyk Extension on the Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner) and click **Install**.\
-  For more information see the [installation instructions](https://code.visualstudio.com/docs/editor/extension-marketplace#\_install-an-extension).
-* Configure the Snyk CLI (downloaded when the extension in installed); see [Visual Studio Code extension configuration](https://docs.snyk.io/ide-tools/visual-studio-code-extension-for-snyk-code/visual-studio-code-extension-configuration).
-* Authenticate with Snyk; see [Visual Studio Code extension authentication](https://docs.snyk.io/ide-tools/visual-studio-code-extension-for-snyk-code/visual-studio-code-extension-authentication).
-* Navigate back to the IDE; the first scan starts automatically.
+- Open the settings or preferences in your IDE.
+- Navigate to the [Snyk Extension on the Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner) and click **Install**.\
+  For more information see the [installation instructions](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-an-extension).
+- Configure the Snyk CLI (downloaded when the extension in installed); see [Visual Studio Code extension configuration](https://docs.snyk.io/ide-tools/visual-studio-code-extension-for-snyk-code/visual-studio-code-extension-configuration).
+- Authenticate with Snyk; see [Visual Studio Code extension authentication](https://docs.snyk.io/ide-tools/visual-studio-code-extension-for-snyk-code/visual-studio-code-extension-authentication).
+- Navigate back to the IDE; the first scan starts automatically.
 
 ## Support
 
-If you need help, submit a request to [Snyk Support](https://support.snyk.io/hc/en-us/requests/new).
```

Comment:
@j-luong This will be overwritten in https://github.com/snyk/vscode-extension/pull/358